Integrations/Integration-Samanage.yml

```diff
@@ -0,0 +1,119 @@
+
+    def sendRequest(method, url, uri, user, passwd, data = None):
+        res = None
+        headers = { 'accept' : 'application/vnd.samanage.v1.1+json', 'Content-Type' : 'application/json' }
+
+        if method.lower() == 'get':
+            res = requests.get(url + uri, auth = (user, passwd), data = headers )
+
+        if method.lower() == 'post':
+            try:
+                temp = json.loads(data)
+            except ValueError, e:
+                return '### Error: Data is not in JSON format'
+
+            res = requests.post(url + uri, auth = (user, passwd), data = data)
+
+        if res.status_code >= 400:
+            if res.text.startswith('{'):
+                return '### Error: ' + str(res.status_code) + ': ' + res.json()["error"]
+            else:
+                return '### Error: ' + str(res.status_code) + ': ' + url + uri
+
+        return res.text
+
+    res = []
+
+    username = demisto.params()['username']
+    password = demisto.params()['password']
+    server = demisto.params()['server']
+
+    if demisto.command() == 'test-module':
+        res = sendRequest('get', server, '/catalog_items.json', username, password)
+        if res.startswith('### Error:'):
+            demisto.results( { 'Type' : entryTypes['error'], 'ContentsFormat' : 'text', 'Contents' : res } )
+        elif 'api.samanage.com' in res:
+            demisto.results('ok')
+    else:
+        if demisto.command() == 'sam-hardware':
+            method = 'get'
+
+            param = demisto.args()['param']
+            query = demisto.args()['query']
+
+            uri = '/hardwares.json?' + param + '=' + query
+        if demisto.command() == 'sam-user':
+            method = 'get'
+
+            param = demisto.args()['param']
+            query = demisto.args()['query']
+
+            uri = '/users.json?' + param + '=' + query
+
+        res = sendRequest(method, server, uri, username, password)
+
+        if res.startswith('### Error:'):
+            demisto.results( { 'Type' : entryTypes['error'], 'ContentsFormat' : 'text', 'Contents' : res } )
+            sys.exit(0)
+        elif res.startswith('[{'):
+           res = json.loads(res)
+
+        demisto.results(res)
+
+    sys.exit(0)
+  type: python
+  commands:
+  - name: sam-hardware
+    deprecated: false
+    arguments:
+    - name: param
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - name
+      - mac_address
+      description: What parameter is your query for? Wildcards accepted. Multiple
+        values can be entered separated by commas.
+    - name: query
+      required: true
+    description: Query various databases in samanage
+  - name: sam-user
+    deprecated: false
+    arguments:
+    - name: param
+      required: true
+      auto: PREDEFINED
+      predefined:
+      - name
+      - email
+    - name: query
+      required: true
+    description: Query Samanage for user information
+  dockerimage: aviadlich/python-requests
```

Comment:
you can switch this to demisto/python
Our official image includes python requests